src/scripts/main.js

```diff
@@ -1 +1,78 @@
 'use strict';
+
+const WHITE_CELL_COLOR = 'chess__white';
+const BLACK_CELL_COLOR = 'chess__black';
+const CELL_OF_NUMBER = 'chess__number';
+const CHESS_FIELD = 'chess__field';
+const CHESS_FIELD_OF_NUMBERS_RIGHT = 'chess__column chess__column-right';
+const CHESS_FIELD_OF_NUMBERS_LEFT = 'chess__column chess__column-left';
+const CHESS_LETTERS_UP = 'chess__letters chess__letters-up';
+const CHESS_LETTERS_DOWN = 'chess__letters chess__letters-down';
+const CHESS_LETTER = 'chess__letter';
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8].reverse();
+const letters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+const changeColor = (i, j) => (
```

Comment:
This function doesn't change any. Can you rename it to `getColorClassname`, for example?